boston_housing/mlcube/.mlcube.yaml

```diff
@@ -0,0 +1,29 @@
+name: MLCommons Boston Housing
```

Comment:
I don't follow why this is called ".mlcube.yaml"; whats the motivation for prefixed "."?

Reply:
The ".mlcube.yaml" file corresponds to the packed version for the MLCube definition, this file contains the needed information to unpack the full MLCube original structure folders and files using the command: `mlcube describe`. This will also generate a file called "mlcube.yaml" (without the initial dot).

I'm currently working on the implementation of config 2.0 for this tutorial, when this is done we won't need to extract the MLCube original folder structure, and instead, there will be only a single file called: "mlcube.yaml" with all the needed information.